packages/integration/src/gerrit/core.test.ts

```diff
@@ -37,15 +37,18 @@ describe('gerrit core', () => {
   describe('buildGerritGitilesArchiveUrl', () => {
     const config: GerritIntegrationConfig = {
       host: 'gerrit.com',
+      baseUrl: 'https://gerrit.com',
       gitilesBaseUrl: 'https://gerrit.com/gitiles',
     };
     const configWithPath: GerritIntegrationConfig = {
       host: 'gerrit.com',
-      gitilesBaseUrl: 'https://gerrit.com/path/gitiles',
+      baseUrl: 'https://gerrit.com/gerrit',
+      gitilesBaseUrl: 'https://gerrit.com/gerrit/plugins/gitiles',
     };
-    const configWithPathSlash: GerritIntegrationConfig = {
+    const configWithDedicatedGitiles: GerritIntegrationConfig = {
```

Comment:
Could we keep a test case that tests for trailing slashes? Just in case

Reply:
Are you sure? Slashes are trimmed in the config-reader, so it wont reflect a code path we expect to happen.

Reply:
We actually don't handle extra slashes in `getGitilesAuthenticationUrl`. Do you want extra logic for that, even though we throw them away much earlier?

Reply:
Alright we can skip it